cmake/script/GenerateHeaderFromRaw.cmake

```diff
@@ -0,0 +1,22 @@
+# Copyright (c) 2023-present The Bitcoin Core developers
+# Distributed under the MIT software license, see the accompanying
+# file COPYING or https://opensource.org/license/mit/.
+
+file(READ ${RAW_SOURCE_PATH} hex_content HEX)
+string(REGEX MATCHALL "([A-Za-z0-9][A-Za-z0-9])" bytes "${hex_content}")
```

Comment:
Wanted to suggest`"([A-Za-z0-9]{2})"` (or `\w{2}`), but it seems the regex parser was written almost 40 years ago https://github.com/Kitware/CMake/blob/master/Source/kwsys/RegularExpression.cxx#L628

Reply:
Yes. The CMake [Regex Specification](https://cmake.org/cmake/help/latest/command/string.html#regex-specification) does not include `\w` support.